src/main/java/hw_9/task1/Earth.java

```diff
@@ -0,0 +1,11 @@
+package hw_9.task1;
+
+public class Earth implements Planet {
+    private final String name = "Earth";
```

Comment:
1) constants should be in upper case - NAME
2) generally constants re static - private static final String <name> = ....

fix in all places please

Reply:
fixed